Core/include/Acts/EventData/MultiTrajectory.ipp

```diff
@@ -93,55 +93,48 @@ inline auto TrackStateProxy<SL, N, M, ReadOnly>::predictedCovariance() const
   return Covariance(m_traj->m_cov.col(data().ipredicted).data());
 }
 
-template <typename SL, size_t N, size_t M, bool ReadOnly>
-inline BoundParameters TrackStateProxy<SL, N, M, ReadOnly>::predictedParameters(
-    const Acts::GeometryContext& gctx) const {
-  return {gctx, predictedCovariance(), predicted(),
-          m_traj->m_referenceSurfaces[data().irefsurface]};
-}
-
 template <typename SL, size_t N, size_t M, bool ReadOnly>
 inline auto TrackStateProxy<SL, N, M, ReadOnly>::filtered() const
     -> Parameters {
   assert(data().ifiltered != IndexData::kInvalid);
   return Parameters(m_traj->m_params.col(data().ifiltered).data());
 }
 
+template <typename SL, size_t N, size_t M, bool ReadOnly>
+inline auto TrackStateProxy<SL, N, M, ReadOnly>::filtered(
+    const Acts::GeometryContext& gctx) const -> FreeVector {
```

Comment:
The only thing that differentiates between this method and the other `filtered()` method in the overload resolution is the argument. Can we change the name of these free parameter getters to something like `filteredGlobal`, or `filteredFree` et al?

Reply:
Of course, I will change it

Reply:
I agree with @paulgessinger. This should be named `filteredFree` or `freeFiltered`. My personal preference is on the latter as it groups the different accessors nicer together, i.e. `free{Filtered,Smoothed,...}`.

Reply:
Changed to `freeXy`